Libs/MRML/Core/vtkMRMLScalarVolumeDisplayNode.cxx

```diff
@@ -796,7 +796,7 @@ void vtkMRMLScalarVolumeDisplayNode::CalculateAutoLevels()
 
     window = max-min;
     level = 0.5*(max+min);
-    lower = this->GetLevel();
+    lower = range[0];
```

Comment:
@agirault yes, the default was not too important so nobody noticed - the option was only ever turned on in the GUI and the slider is right there to adjust (which is almost always needed in practice no matter what default we pick).